application/views/survey_responses_view.py

```diff
@@ -44,10 +45,12 @@ def add_survey_response(case_id):
 
         upload_success = survey_response.add_survey_response(case_id, file, file_name, survey_ref)
 
+        if upload_success == "False":
+            return make_response(FILE_TOO_SMALL, 400)
+
         if upload_success:
```

Comment:
I'd be tempted to move the error variables into the survey response and then do something like

upload_success, error = survey_response.add_survey_response(case_id, file, file_name, survey_ref)

if upload_success:
  return make_response(UPLOAD_SUCCESSFUL, 200)
else:
     return make_response(error, 500)